src/plugins/es_ui_shared/static/forms/hook_form_lib/hooks/use_form.ts

```diff
@@ -18,7 +18,7 @@ const DEFAULT_OPTIONS = {
   stripEmptyFields: true,
 };
 
-interface UseFormReturn<T extends FormData, I extends FormData> {
+export interface UseFormReturn<T extends FormData, I extends FormData> {
```

Comment:
@patrykkopycinski I grepped but couldn't find where this is being used. Does this need to be exported?

Reply:
@cjcenizal I do return it here https://github.com/elastic/kibana/pull/101835/files#diff-ef686a1c10d09bc8768626765f75a3abae956994d673229c1995c4cb927b4531R38 and without `export` I was getting type errors

Reply:
I think @cjcenizal point is valid. You seem to _implicitly_ need this export. I think it would be good to be explicit.

So export it from the `index.ts` of the "hooks" folder and then export it in the public index.ts barrel of the "hook_form_lib" folder.

And then explicitly set the export of your `useScheduledQueryGroupQueryForm()` handler. We will then be able to trace where the interface is being used. 👍